### PR TITLE
Add lifecycle events documentation. Related (and hopefully fixes) #12

### PR DESCRIPTION
**Pull request by danigb**
Add some  documentation to README.md with a `canvas` example.

### CODE REVIEW

**Comment by jbucaran** [comment]
@danigb Very nice! Thanks, what's `...`?

**Comment by danigb** [comment]
Great! The `...` means `insert your code here`, but maybe I should remove it, isn't it? 😂

**Comment by jbucaran** [comment]
@danigb LOL, right. Can we have simple working example?

File: README.md [hunk 1]
```diff
@@ -339,6 +340,51 @@ app({
...
+- `onremove(e : HTMLElement)`
+
+The lifecycle event handler receives a reference to the DOM element.
+
+<details>
+<summary><i>Example</i></summary>
+
+```js
+function repaint(canvas, model) {
+  var ctx = canvas.getContext('2d')
```
Comment (maraisr) [inline README.md]: Could you please spell this out, `context` i know its a pain, but to most people ctx means nothing! Love the contrib though! :heart:

**Comment by danigb** [comment]
With a working example now

File: README.md (hunk 1 above)
Comment (danigb) [inline README.md]: Sorry, I didn't read you comment. Fixing it...

**Comment by jbucaran** [comment]
@danigb Awesome, thanks!

**Pull request merged**